docker/reference/reference.go

```diff
@@ -102,13 +107,13 @@ func WithTag(name Named, tag string) (NamedTagged, error) {
 
 // WithDigest combines the name from "name" and the digest from "digest" to form
 // a reference incorporating both the name and the digest.
-func WithDigest(name Named, digest digest.Digest) (Canonical, error) {
-	r, err := distreference.WithDigest(name, digest)
-	if err != nil {
-		return nil, err
-	}
-	return &canonicalRef{namedRef{r}}, nil
-}
+//func WithDigest(name Named, digest digest.Digest) (Canonical, error) {
```

Comment:
This happens because `GOPATH=vendor/...` breaks everything. As someone who does a lot of symlink hacks to make Go behave, I learn this first hand far more often than most people (who allow Google to define how their harddrive is organised).